<launch>

  <arg name="models" default="/home/v4r/models" doc="Models directory"/>
  <arg name="config" default="/home/v4r/catkin_ws/src/ppf_recognizer_ros/cfg"
       doc="Object Recognition XML configuration directory"/>
  <arg name="args" default=""
       doc="Additional arguments as stated in the library. For help args:=-h."/>

  <group ns="hsr_grasping">
    <node pkg="ppf_recognizer_ros" type="recognition_service"
          name="recognition_service" output="screen" args="$(arg args)">

      <!-- remaps from /hsr_grasping/recognition_service/estimate_poses to
            /hsr_grasping/estimate_poses-->
      <remap from="~estimate_poses" to="estimate_poses"/>

      <param name="models_dir" value="$(arg models)"/>
      <param name="cfg_dir" value="$(arg config)"/>

    </node>
  </group>

</launch>
